modules/social_features/social_group/social_group.tokens.inc

```diff
@@ -190,7 +190,7 @@ function social_group_tokens_alter(array &$replacements, array $context, Bubblea
                 case 'join_to_group':
                   if (isset($context['tokens']['cta_button'])) {
                     $link = Url::fromRoute('view.group_members.page_group_members', ['group' => $group_content->getGroup()->id()]);
-                    $cta_button = $email_token_services->getCtaButton($link, new TranslatableMarkup('See all members'));
+                    $cta_button = $email_token_services->getCtaButton($link, new TranslatableMarkup('See all members', [], $context['options']));
```

Comment:
@volodymyr-sydor just noticed that we change `getCtaButton()` method only in one place, but we have multiple references of this method. Shouldn't we update all of them?
<img width="962" alt="image" src="https://github.com/goalgorilla/open_social/assets/2241917/808bf56a-99fa-4f5e-9dbf-9efdb8fe616c">